torchprime/launcher/save_hf_tokenizer_and_model.py

```diff
@@ -0,0 +1,94 @@
+"""Utilities for preparing Hugging Face assets (models and tokenizers) for GCS."""
+
+import logging
+import os
+import subprocess
+import tempfile
+from pathlib import Path
+
+from huggingface_hub import snapshot_download
+
+logger = logging.getLogger(__name__)
+
+TOKENIZER_PATTERNS = [
```

Comment:
nit for constants it is recommended to use immutable data types, tuple in this case:
TOKENIZER_PATTERNS = (a, b, c,)